src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -581,6 +581,21 @@
     AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String text);
 
     /**
+     * Analyze texts under the provided index.
+     *
+     * @param index The index name
+     * @param text  The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String... text);
```

Comment:
Good point

I think we should drop the `prepareAnalyze(String... text)` arity, and force specifying the index if you want to use the varargs analysis. What do you think?